sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollationExpressionSuite.scala

```diff
@@ -59,7 +59,9 @@ class CollationExpressionSuite extends SparkFunSuite with ExpressionEvalHelper {
 
   test("collate on non existing collation") {
     checkError(
-      exception = intercept[SparkException] { Collate(Literal("abc"), "UCS_BASIS") },
+      exception = intercept[SparkException] {
```

Comment:
To make your life easier when merging the change further on, please avoid needless changes :)